pool/app/session/entity.ml

```diff
@@ -521,6 +521,13 @@ let not_closed session =
     |> CCResult.fail
 ;;
 
+let is_not_closed_nor_canceled session =
```

Comment:
what's the difference to code line [559](https://github.com/uzh/pool/pull/239/files#diff-345ed91fcdef2c2963bd83c5cce85b4f3352f9f9bb897b6bc22030e3647f281fR559)

Reply:
missed that, fixed